pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1078,7 +1078,7 @@ def test_dt64arr_add_dtlike_raises(self, tz_naive_fixture, box_with_array):
     # Note: freq here includes both Tick and non-Tick offsets; this is
     #  relevant because historically integer-addition was allowed if we had
     #  a freq.
-    @pytest.mark.parametrize("freq", ["H", "D", "W", "M", "MS", "Q", "B", None])
+    @pytest.mark.parametrize("freq", ["H", "D", "W", "ME", "MS", "Q", "B", None])
```

Comment:
maybe we should include "2ME" in here, bit worried we're not handling that

Reply:
It’s a good idea, I did it. Seems after replacing `“ME”` with `“2ME”` we don’t have failures in tests.